From 9cc622bd07227e1a3777c4be43211d4b37370e06 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Tue, 22 May 2012 16:03:56 +0800
Subject: [PATCH 1535/2463] ENGR00210075-4 SPDC: Export some APIs for unit
 test case.

- Export some APIs for unit test driver use.

Signed-off-by: Fugang Duan  <B38611@freescale.com>
---
 drivers/video/mxc/mxc_spdc_fb.c |    6 ++++--
 1 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/video/mxc/mxc_spdc_fb.c b/drivers/video/mxc/mxc_spdc_fb.c
index 4ae8bf6..37938b9 100644
--- a/drivers/video/mxc/mxc_spdc_fb.c
+++ b/drivers/video/mxc/mxc_spdc_fb.c
@@ -2381,7 +2381,7 @@ pxp_process:
 	mutex_unlock(&fb_data->queue_mutex);
 }
 
-static int mxc_spdc_fb_send_update(struct mxcfb_update_data *upd_data,
+int mxc_spdc_fb_send_update(struct mxcfb_update_data *upd_data,
 				   struct fb_info *info)
 {
 	mxc_spdc_t *fb_data = info ?
@@ -2603,11 +2603,12 @@ static int mxc_spdc_fb_send_update(struct mxcfb_update_data *upd_data,
 	mutex_unlock(&fb_data->queue_mutex);
 	return 0;
 }
+EXPORT_SYMBOL(mxc_spdc_fb_send_update);
 
 /*
  * return 0 : spdc update is update
  */
-static int
+int
 mxc_spdc_fb_wait_update_complete(struct mxcfb_update_marker_data *marker_data,
 				struct fb_info *info)
 {
@@ -2669,6 +2670,7 @@ mxc_spdc_fb_wait_update_complete(struct mxcfb_update_marker_data *marker_data,
 
 	return ret;
 }
+EXPORT_SYMBOL(mxc_spdc_fb_wait_update_complete);
 
 int mxc_spdc_fb_set_pwrdown_delay(u32 pwrdown_delay,
 				struct fb_info *info)
-- 
1.7.7.4

